example/exampleLineCrossSection.js

```diff
@@ -209,23 +209,35 @@ function create_charts(
         }
       }
 
+      let is_cross_section_enabled = false
```

Comment:
this is an example of how you can determine if a cross-section line that is deleted was deleted as a result of a new line draw.